### ISSUE TITLE
NTR: Mononeuritis multiplex

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#3981]
New term request: Mononeuritis multiplex. Synonyms: mononeuropathy multiplex, multifocal neuropathy\
Definition: A type of peripheral neuropathy that happens when there is damage to two or more different nerve areas. It is is a group of symptoms rather than its own disease. \
Source: A type of peripheral neuropathy that happens when there is damage to two or more different nerve areas. It is is a group of symptoms rather than its own disease. \
Parent: http://www.aanem.org/Patients/Disorders/Mononeuritis-Multiplex\
Disease association: Mononeuropathy

**Comment by pnrobinson** [comment]
- [ ] Should this be a child of Mononeuropathy https://hpo.jax.org/app/browse/term/HP:0009831

- [ ] We should describe how this phenotype is measured/observed. For instance is it with  NCS/EMG 

Comment: Mononeuritis multiplex (MM) can be a manifestation of several disorders including infectious, inflammatory, neoplastic, toxic, metabolic and hereditary conditions, and the etiological diagnosis may be challenging. (from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4275932/)

**Comment by pnrobinson** [comment]
Make sib of Mononeuropathy 
Def -- can be purely clinical

**Comment by pnrobinson** [comment]
Multiple mononeuropathy should be the primary label (see https://www.ncbi.nlm.nih.gov/pubmed/19015303)
